ssl/quic/quic_record_shared.c

```diff
@@ -150,8 +150,10 @@ static int el_setup_keyslot(OSSL_QRL_ENC_LEVEL_SET *els,
         goto err;
```

Comment:
We're calling `tls13_hkdf_expand_ex` above with `raise_err` as 0 to suppress error raising...but below we're adding explicit error raising - which seems inconsistent. Also there are a number of cases above where we just "return 0" and don't  raise an error.

Reply:
I've changed the tls13_hkdf_expand_ex() calls throughout the code to raise errors.